buildCompleteWebsite.sh

```diff
@@ -26,31 +26,31 @@ mkdir -p build
 cd build
 if [ -d "xeps" ]; then
 	cd xeps
-	git pull
+	git pull || exit 1;
```

Comment:
It can, but IMO if your doing that inside of a CI script then you have other issues.

For that pattern then `trap "exit 1" EXIT` works